searchlib/src/tests/sortspec/multilevelsort.cpp

```diff
@@ -431,4 +414,45 @@ TEST("test that [docid] translates to [lid][paritionid]") {
     EXPECT_EQUAL(0, memcmp(SECOND_DESC, sr2.first, 6));
 }
 
+TEST("test that [docid] uses attribute when one exists") {
+    vespalib::TestClock clock;
```

Comment:
Perhaps some of the setup code could be refactored and shared across different test cases?

Reply:
Yes, I could benefit from some extra care this code. Will keep in mind.